scene/gui/control.cpp

```diff
@@ -2377,12 +2367,7 @@ Control *Control::_get_focus_neighbour(Margin p_margin, int p_count) {
 		Node *n = get_node(data.focus_neighbour[p_margin]);
 		if (n) {
 			c = Object::cast_to<Control>(n);
-
-			if (!c) {
-
-				ERR_EXPLAIN("Neighbour focus node is not a control: " + n->get_name());
-				ERR_FAIL_V(NULL);
-			}
+			ERR_FAIL_COND_V_MSG(!c, NULL, "Neighbour focus node is not a control: " + n->get_name() + ".");
```

Comment:
en_GB spotted, should be `Neighbor`.

Reply:
The method itself has "neighbour" in name...

Reply:
Oops :D Probably one to list in #16863.